### PR TITLE
Updated ADMIN_USERNAME to indicate default value, Project Owner email…

### PR DESCRIPTION
**Pull request by meker12**
## This PR is a:
- [X] Content fix or rewrite

 

## Summary
 
Update Magento Cloud onboarding information and environment variables to indicate that the ADMIN_USERNAME environment variable default value is the Project Owner email address.

### CODE REVIEW

File: guides/v2.1/cloud/env/environment-vars_magento.md [hunk 1]
```diff
@@ -44,7 +44,11 @@ The following environment variables are available for overriding administrative
     </tr>
     <tr>
       <td><code>ADMIN_USERNAME</code></td>
-      <td>Username for a Magento administrative user. This user is an administrator and can create other users, including other administrative users.</td>
+      <td>User name for a Magento administrative user. This user is an
+      administrator and can create other users, including other administrative
+      users. When you create a Magento trial account, the ADMIN_USERNAME value
+      defaults to the Project Owner email address. This value can be changed
```
Comment (hguthrie) [inline guides/v2.1/cloud/env/environment-vars_magento.md]: "can be changed" ...Maybe "You can change..."

File: guides/v2.1/cloud/onboarding/onboarding-tasks.md [hunk 2]
```diff
@@ -24,7 +24,10 @@ If you joined us with a 30-day free trial {{site.data.var.ece}} account, we comp
 -   Provisioned the initial environment with a cloned {{site.data.var.ece}} code base in a `master` branch
 -   Created an environment for the `master` branch with a web server, database, and services
 -   Added Magento authentication keys for the Project Owner
--   Add the `ADMIN_EMAIL` variable. See [Set environment and project variables]({{ page.baseurl }}/cloud/project/project-webint-basic.html#project-conf-env-var).
+-   Added the `ADMIN_EMAIL` and `ADMIN_USERNAME` variables with both set to the
+default value, Project Owner email address.
+[Set environment and project variables]({{ page.baseurl }}/cloud/project/project-webint-basic.html#project-conf-env-var).
```
Comment (hguthrie) [inline guides/v2.1/cloud/onboarding/onboarding-tasks.md]: Maybe use "See" to present the link.

File: guides/v2.1/cloud/onboarding/onboarding-tasks.md [hunk 3]
```diff
@@ -70,6 +73,11 @@ When you initially create your project, you can access the Magento Admin panel t
...
 1.  Click the forgot Password link and complete the process to change the password.
-1.  Login to the Admin panel using the username Admin and newly changed password.
-
-If you receive an error, you may need to add admin variables for `ADMIN_EMAIL`, `ADMIN_USERNAME`, and `ADMIN_PASSWORD`. Trial accounts should already have the `ADMIN_EMAIL` variable configured for them.
\ No newline at end of file
+1.  Login to the Admin panel using the default username (Project Owner email address)
+and the newly changed password.
+
+If you receive an error, you might need to add admin variables for
+`ADMIN_EMAIL`, `ADMIN_USERNAME`, and `ADMIN_PASSWORD`. Trial accounts should
```
Comment (hguthrie) [inline guides/v2.1/cloud/onboarding/onboarding-tasks.md]: This paragraph has some tense conflicts and a wishy washy feel to it, like we're not sure what they will see. The phrase "should already have" is also hard to think about, especially if you're translating it. And there should be a comma before using "which".

File: guides/v2.2/cloud/env/environment-vars_magento.md [hunk 4]
```diff
@@ -44,7 +44,11 @@ The following environment variables are available for overriding administrative
     </tr>
     <tr>
       <td><code>ADMIN_USERNAME</code></td>
-      <td>Username for a Magento administrative user. This user is an administrator and can create other users, including other administrative users.</td>
+      <td>User name for a Magento administrative user. This user is an
+      administrator and can create other users, including other administrative
+      users. When you create a Magento trial account, the ADMIN_USERNAME value
+      defaults to the Project Owner email address. This value can be changed
```
Comment (hguthrie) [inline guides/v2.2/cloud/env/environment-vars_magento.md]: You can change the ADMIN_USERNAME from the Magento Admin panel.

File: guides/v2.1/cloud/before/before-setup-env-2_clone.md [hunk 5]
```diff
@@ -73,7 +73,7 @@ To clone the project's `master` environment to your local:
 We recommend changing the following variables for the Magento Admin URL and administrator account. You should configure these settings for security reasons prior to branching from the cloned `master`. If you change the variables in the `master` branch, you only have to make these changes once. All branches inherit the variables from `master`.
 
 * `ADMIN_EMAIL`: Administrative user's e-mail address. This value is required for upgrading and patching Magento Commerce (Cloud) and is used to send password reset emails.
-* `ADMIN_USERNAME`: Username for a Magento administrative user. This user is an administrator and can create other users, including other administrative users. The default hardcoded username is `admin`. You can use `admin` or change it to another secure username.
+* `ADMIN_USERNAME`: User name for a Magento administrative user. This user is an administrator and can create other users, including other administrative users. The default hardcoded username is the Project Owner email address. You can use this value, or change it to another secure username.
```
Comment (jeff-matthews) [inline guides/v2.1/cloud/before/before-setup-env-2_clone.md]: Shouldn't this be "username"; one word? We're using both forms of the term in this paragraph.

File: guides/v2.1/cloud/onboarding/onboarding-tasks.md [hunk 6]
```diff
@@ -24,7 +24,10 @@ If you joined us with a 30-day free trial {{site.data.var.ece}} account, we comp
 -   Provisioned the initial environment with a cloned {{site.data.var.ece}} code base in a `master` branch
 -   Created an environment for the `master` branch with a web server, database, and services
 -   Added Magento authentication keys for the Project Owner
--   Add the `ADMIN_EMAIL` variable. See [Set environment and project variables]({{ page.baseurl }}/cloud/project/project-webint-basic.html#project-conf-env-var).
+-   Added the `ADMIN_EMAIL` and `ADMIN_USERNAME` variables with both set to the
+default value, Project Owner email address.
```
Comment (jeff-matthews) [inline guides/v2.1/cloud/onboarding/onboarding-tasks.md]: This comma seems out of place... shouldn't it be a colon?

**Pull request merged**